### PR TITLE
isidentifier: Remove Python 2 specific code

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE

- Test Pull Request

### CODE REVIEW

File: lib/ansible/utils/vars.py [hunk 1]
```diff
@@ -265,8 +263,6 @@ def _isidentifier_PY3(ident):
 portable between the two. The following changes were made:
 
     * disallow non-ascii characters (Python 3 allows for them as opposed to Python 2)
-    * True, False and None are reserved keywords (these are reserved keywords
```
Comment (mkrizek) [inline lib/ansible/utils/vars.py:268]: I think we can just rename `_isidentifier_PY3` to `isidentifier` and move this doc string there at this point.

Comment (bcoca) [reply lib/ansible/utils/vars.py:268]: bikeshed: is_identifier, just to match most of our 'is_this' functions/methods, even 'is_python_identifier' so no one needs to jump to read the function to realize what it is.

Comment (mkrizek) [reply lib/ansible/utils/vars.py:268]: "rename" wasn't precise. `isidentifier` already exists and is a public interface. We just assigned `_isidentifier_PY3` or `_isidentifier_PY2` to it depending on the Python version. Now that only `_isidentifier_PY3` remains it is no longer needed.

**Pull request merged**